### PR TITLE
Aligned search-bar with the search-button.

### PR DESCRIPTION
**Pull request by ankur12-1610**
Fixes: #780 

Before:
search-bar

After:
search-solved

### CODE REVIEW

**Comment by ankur12-1610** [comment]
I have used style instead of making a new CSS class and changing the class in  `www/bootstrap.css` might create a problem :)

**Comment by Jaifroid** [comment]
Working on IE11.

**Review by Jaifroid — CHANGES_REQUESTED** [review]
The style change should be made in app.css, not added to index.html. Please either edit an existing style rule in app.css or else add a new rule if there is no appropriate rule you can edit.

**Comment by Jaifroid** [comment]
> I have used style instead of making a new CSS class and changing the class in `www/bootstrap.css` might create a problem :)

I agree that you must not edit `bootstrap.css`. However, we have our own stylesheet which can be used to override boostrap styles if necessary.

**Comment by ankur12-1610** [comment]
@Jaifroid done the required changes, kindly review :')

File: www/index.html [hunk 1]
```diff
@@ -82,8 +82,6 @@
                                     <input type="search" id="prefix" placeholder="Search..." class="form-control" />
                                     <button type="button" class="btn btn-outline-secondary" id="searchArticles"><i class="fas fa-search"></i></button>
                                 </span>
-                            </span>
-                            <span class="col-2">
```
Comment (Jaifroid) [inline www/index.html:86]: @ankur12-1610 Please explain why you have deleted the last two columns of the Bootstrap 12-column grid. You have effectively created a line that only uses 10 of the 12 columns.

**Comment by Jaifroid** [comment]
Just as a general comment on procedure, please don't create a new branch when you revise a PR. Simply push your changes to the existing branch, so that we can keep a history of the changes you have made over the various commits.

File: www/index.html (hunk 1 above)
Comment (ankur12-1610) [reply www/index.html]: > @ankur12-1610 Please explain why you have deleted the last two columns of the Bootstrap 12-column grid. You have effectively created a line that only uses 10 of the 12 columns.

Oh! sorry I guess it was by mistake I didn't check the files changed, sorry for this one :')

Comment (ankur12-1610) [reply www/index.html]: > @ankur12-1610 Please explain why you have deleted the last two columns of the Bootstrap 12-column grid. You have effectively created a line that only uses 10 of the 12 columns.

I've amended the changes, please look into it.

**Comment by ankur12-1610** [comment]
> Just as a general comment on procedure, please don't create a new branch when you revise a PR. Simply push your changes to the existing branch, so that we can keep a history of the changes you have made over the various commits.

Yeah sure.
But how can I work on multiple issues? :thinking: 
Generally I do rebase the issue branch on master branch if any commits are merged, that solves the problem. Is this solution valid?

**Comment by Jaifroid** [comment]
You just need to create different branches for each PR you are working on, and switch between them when you decide to work on a different one. Usually there will be no conflicts, but if there are, you use conflict resolution when you or we squash and merge into master. This way you keep your work on each issue separate, and the commits are clean.

**Comment by ankur12-1610** [comment]
> You just need to create different branches for each PR you are working on, and switch between them when you decide to work on a different one. Usually there will be no conflicts, but if there are, you use conflict resolution when you or we squash and merge into master. This way you keep your work on each issue separate, and the commits are clean.

Yeah, but @Jaifroid like I'm doing the same thing right? :thinking: Creating different branches for different issues. Or is there something wrong with my current way?

**Comment by Jaifroid** [comment]
> Yeah, but @Jaifroid like I'm doing the same thing right? 🤔 Creating different branches for different issues. Or is there something wrong with my current way?

You are currently rebasing and force-pushing changes to this PR. If you have to force-push, then you are definitely doing something wrong! A force-push should be reserved, very rarely, for correcting a serious mistake. When you simply make incremental changes to a PR, you should simply push the changes. If you force-push, then you force me to reset my version with a hard reset before I can review and test your code locally.

When you want to work on a different issue, all you have to do is commit any changes you have made to the current branch, and switch branches.

**Comment by ankur12-1610** [comment]
> > Yeah, but @Jaifroid like I'm doing the same thing right? thinking Creating different branches for different issues. Or is there something wrong with my current way?
> 
> You are currently rebasing and force-pushing changes to this PR. If you have to force-push, then you are definitely doing something wrong! A force-push should be reserved, very rarely, for correcting a serious mistake. When you simply make incremental changes to a PR, you should simply push the changes. If you force-push, then you force me to reset my version with a hard reset before I can review and test your code locally.
> 
> When you want to work on a different issue, all you have to do is commit any changes you have made to the current branch, and switch branches.

Ah! got it! I'm really sorry for it, I force pushed to avoid writing new commit messages, which is wrong I realized. I'm really sorry for force pushes it won't happen next time. 

So now do I need to add a new commit or do I need to close the PR and create a new one?

**Comment by Jaifroid** [comment]
No problem @ankur12-1610, we've all had to learn the same way!
No need to close and start a new PR, I've now reset my copy to the rebased branch, so will test shortly.

**Comment by ankur12-1610** [comment]
> No problem @ankur12-1610, we've all had to learn the same way! No need to close and start a new PR, I've now reset my copy to the rebased branch, so will test shortly.

Thanks! a lot :)

**Review by Jaifroid — APPROVED** [review]
This is working for me, thank you @ankur12-1610 . @mossroy would you be happy for a merge?

**Comment by ankur12-1610** [comment]
> Working on IE11, and also working with very small screen dimensions.
> 
> image

Great :raised_hands:

**Comment by mossroy** [comment]
Yes, you can squash and merge

**Pull request merged**